easybuild/easyconfigs/m/METIS/METIS-5.1.0-foss-2017a.eb

```diff
@@ -0,0 +1,22 @@
+name = 'METIS'
+version = '5.1.0'
+
+homepage = 'http://glaros.dtc.umn.edu/gkhome/metis/metis/overview'
+description = """METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes,
+ and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the
+ multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis',
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD',
+]
```

Comment:
@henkela for recent `METIS` easyconfigs, we include the `METIS-5.1.0-use-doubles.patch` patch (double-precision floating-point), this is required when METIS is used as a dep for OpenFOAM 4